packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1090,6 +1094,216 @@ describe("SharedTree", () => {
 				});
 			}
 		});
+
+		describe("can resubmit", () => {
+			const sb = new SchemaBuilder({ scope: "shared tree undo tests" });
+			const schema = sb.intoSchema(sb.list(sb.list(sb.string)));
+
+			/**
+			 * Each test will involve two peers, one of which has removed a tree (and is therefore able to restore it) -
+			 * let's call that one peer 1.
+			 *
+			 * Peer 1 can do one of three things:
+			 * (A) Commit 1: make an edit to the removed tree, Commit 2: make another edit to removed tree
+			 * (B) Commit 1: make an edit to the removed tree, Commit 2: restore the removed tree
+			 * (C) Commit 1: restore the removed tree, Commit 2: make an edit to the restored tree
+			 * Peer 2 can only do option (A) above.
+			 */
+			const scenarios = [
```

Comment:
The doc comment on this list of scenarios is confusing because the items in the list do not represent `[whatPeer1Does, whatPeer2Does]` which is what one would expect from the comment.
Maybe add a sentence at the end of the doc comment to explain the format here.